dockerfiles/alpine/is/Dockerfile

```diff
@@ -17,9 +17,9 @@
 # ------------------------------------------------------------------------
 
 # set base Docker image to Alpine Docker image
-FROM alpine:3.16.0
+FROM alpine:3.16
 LABEL maintainer="WSO2 Docker Maintainers <dev@wso2.org>" \
-      com.wso2.docker.source="https://github.com/wso2/docker-is/releases/tag/v6.1.0.1"
+      com.wso2.docker.source="https://github.com/wso2/docker-is/releases/tag/v6.1.0.4"
```

Comment:
next version should be v6.1.0.3

Reply:
v6.1.0.3 is already added with https://github.com/wso2/docker-is/pull/389 but the release tag has not been created.